### PR TITLE
Update introduction to best practices

### PR DESCRIPTION
**Pull request by core-man**

### CODE REVIEW

**Comment by core-man** [comment]
Shall we add a first order content?

File: source/best-practices/index.rst [hunk 1]
```diff
@@ -1,4 +1,10 @@
 前言
 ====
 
-这一部分介绍我们在日常科研工作中总结的一些经验，供读者参考。
+这一部分介绍我们在日常科研工作中整理和总结的一些经验，供读者参考。
+欢迎地震学新手在实践中尝试这些经验总结，并再次整理、总结和分享。
```
Comment (core-man) [inline source/best-practices/index.rst:5]:
```suggestion

欢迎地震学新手在实践中尝试这些经验总结，并再次整理、总结和分享。
```

File: source/best-practices/index.rst [hunk 2]
```diff
@@ -1,4 +1,11 @@
 前言
 ====
 
-这一部分介绍我们在日常科研工作中总结的一些经验，供读者参考。
+这一部分介绍我们在日常科研工作中整理和总结的一些经验，供读者参考。
+
+欢迎地震学新手在实践中尝试这些经验总结，并再次整理、总结和分享。
+最后形成实践、整理、总结、再实践、再整理、再总结……的良性循环。
```
Comment (seisman) [inline source/best-practices/index.rst:7]: Use ……

**Comment by seisman** [comment]
> Shall we add a first order content?

Yes to me.

File: source/best-practices/index.rst (hunk 2 above)
Comment (seisman) [reply source/best-practices/index.rst:7]: Never mind. It's Chinese "……", but looks like English "......".

File: source/best-practices/index.rst [hunk 3]
```diff
@@ -1,4 +1,11 @@
...
+
+**实践是检验真理的唯一标准！**
+
+**整理和总结是提高科研生产率的有效催化剂~**
```
Comment (seisman) [inline source/best-practices/index.rst:11]: Not sure what the "quote" syntax look like.
```suggestion
> 实践是检验真理的唯一标准！

> 整理和总结是提高科研生产率的有效催化剂！
```

Comment (core-man) [reply source/best-practices/index.rst]: It seems that the quote does not work.

Comment (seisman) [reply source/best-practices/index.rst]: Please try changing > to |

Comment (core-man) [reply source/best-practices/index.rst]: use four space indetation and ** instead. see 5bda85f.

Comment (core-man) [reply source/best-practices/index.rst]: > Please try changing > to |

｜does not work either.

**Review by seisman — APPROVED** [review]

**Pull request merged**